serverless.yml

```diff
@@ -10,6 +10,9 @@ provider:
   stage: ${opt:stage, 'dev'}
   runtime: python2.7
   timeout: 300
+  environment:
+      AV_DEFINITION_S3_BUCKET: ${env:AV_DEFINITION_S3_BUCKET}
+      DATADOG_API_KEY: ${env:DATADOG_API_KEY}
```

Comment:
You could make this env var "optional" by adding a default blank value like `DATADOG_API_KEY: ${env:DATADOG_API_KEY, ""}`